scripts/globals/damage.lua

```diff
@@ -82,7 +82,7 @@ xi.damage.returnDamageTakenMod = function(target, attackType, damageType)
     then
         dmgTakenMod = dmgTakenMod * ((target:getMod(dmgMods[damageType])) / 1000)
     elseif damageType and dmgMods[damageType] then -- This is for elemental SDTs only
-        dmgTakenMod = dmgTakenMod + (target:getMod(dmgMods[damageType]) / 10000)
+        dmgTakenMod = dmgTakenMod * (1 + (target:getMod(dmgMods[damageType]) / 10000))
```

Comment:
What does this sneaky 1+ do?

Reply:
> What does this sneaky 1+ do?

The 1 + is needed because just adding these mod directly does not work. For instance if you have both a UDMGMAGIC of -5000 and a SDT of -9000 then the dmgTakenMod would be 0.5 (after processing UDMGMAGIC) and then -0.4 (after processing the SDT). Thus you are now incorrectly absorbing damage, instead of the correct mod of 0.50 * 0.10 = 0.05.

Thus we first need to convert the -10000 to 10000 range (with 0 being no change in damage) to something that can be multiplied with other mods (in this case to 0 to 2 with 1 being no change in damage). The 1 representing no change is important because we are multiplying.